test/IRGen/condfail.sil

```diff
@@ -22,23 +23,29 @@ import Swift
 // CHECK-OPT-linux:   ##NO_APP
 // CHECK-OPT-windows:   ##APP
 // CHECK-OPT-windows:   ##NO_APP
+// CHECK-OPT-android:   @APP
+// CHECK-OPT-android:   @NO_APP
```

Comment:
Really?  android aarch64 has the same leader?  IIRC, they had different comment leaders.

Reply:
That's because I though `android` will work, but it is not actually matching anything. Thanks for catching this.

Now I’m not sure that the other prefixes are actually working, though, because `linux` might need to be `linux-gnu`/`linux-gnueabihf` and `windows` might need to be `windows-msvc`.